locale/en/LC_MESSAGES/django.po

```diff
@@ -5653,3 +5647,13 @@ msgstr "Task edited"
 #: tasks/views/task_views.py:50
 msgid "Taak verwijderd"
 msgstr "Task deleted"
+
+#, python-format
+#~ msgid ""
+#~ "\n"
+#~ "                        Revisie/amendement van<br/> %(parent)s\n"
+#~ "                    "
+#~ msgstr ""
+#~ "\n"
+#~ "Revision or amendment of<br/> %(parent)s\n"
+#~ "   "
```

Comment:
Small thing; this PR removes a msgid, so makemessages moves it here commented out.

It does no harm, but it's better to just remove it from the file completely. Otherwise we'd get a large(r) file with unused stuff at the end.